src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -86,15 +103,15 @@ public abstract partial class ComponentBase : Microsoft.AspNetCore.Components.IC
     {
         public ComponentBase() { }
         protected virtual void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTree.RenderTreeBuilder builder) { }
-        protected System.Threading.Tasks.Task Invoke(System.Action workItem) { throw null; }
+        protected System.Threading.Tasks.Task InvokeAsync(System.Action workItem) { throw null; }
```

Comment:
There are a few small API deltas that are riding along with this change. Most of these don't impact the compiler.